torch/csrc/jit/codegen/cuda/transform_iter.h

```diff
@@ -44,14 +44,21 @@ struct Exprs : public IterVisitor {
 
 } // namespace
 
+// Uses the history of _target_domain, and replays that history using the
```

Comment:
reformat this comment to make the individual notes more explicit? (just split it into multiple paragraphs should do the trick)

Reply:
Done, lets see if it gets through clang_format